frontend/crd/src/app/milestones-page/milestones/milestones.component.html

```diff
@@ -1,28 +1,49 @@
 
-<ul>
-  <li *ngFor="let milestone of milestones">
-    <div class="title-container" (click)="milestone.expanded = !milestone.expanded">
-      <span class="title">{{ milestone.name }}</span>
-      <span class ="arrow" [ngClass]="{ 'arrow-up': milestone.expanded, 'arrow-down': !milestone.expanded }"></span>
-    </div>
-    <ng-container *ngIf="milestone.expanded">
-      <div class="content">{{ milestone.yearLevel }}</div>
-      <ng-container *ngFor="let task of milestone.tasks">
-        <div class="content">
-          {{task.description}}
-        </div>
-      </ng-container>
 
-      <ng-container *ngFor="let event of milestone.events">
-        <div class="content">
-          <div>{{event.name}}</div>
-          <div >{{event.description}}</div>
-          <div >{{event.date | date:'short'}}</div>
-          <div >{{event.organizer}}</div>
-          <div >{{event.location}}</div>
-        </div>
-      </ng-container>
+<div class="year-container" *ngFor="let yearLevel of yearLevels">
```

Comment:
right now this entire element shows up on the dashboard as well as the milestones page. is this a placeholder on the dashboard page? shouldn't that space be for upcoming/incomplete tasks and not listing everything?

Reply:
we could possibly keep it on the dashboard since it might be easier to have a similar looking/functioning widget on there until we develop the task notif thing, but at least make a note somewhere that it's just a placeholder

Reply:
should I remove it from the portfolio page or will we want to reuse it there? It looks like John moved the Milestones to the left side of the page, but I think its going to just be a listing of your completed milestones

Reply:
Yeah I think what we'll be showing on the portfolio page will be very different. We don't need to show any individual task or event, just the full milestone